### ISSUE TITLE
IOS: getting errors in Xcode after upgrade

### ISSUE THREAD

**Issue by fbazbo** [LuisThein/platform_maps_flutter#23]
```Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    /Users/me/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/apple_maps_flutter-0.1.2+4/ios/Classes/Annotations/AnnotationController.swift:114:39: error: unterminated string literal
    \(CGFloat(annotation.calloutOffset.y))")
                                          ^
    /Users/me/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/apple_maps_flutter-0.1.2+4/ios/Classes/Annotations/AnnotationController.swift:114:2: error: invalid component of Swift key path
    \(CGFloat(annotation.calloutOffset.y))")
     ^
    /Users/me/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/apple_maps_flutter-0.1.2+4/ios/Classes/Annotations/AnnotationController.swift:114:1: error: expression type 'WritableKeyPath<_, _>' is ambiguous without more context
    \(CGFloat(annotation.calloutOffset.y))")
    ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /Users/me/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/apple_maps_flutter-0.1.2+4/ios/Classes/Annotations/AnnotationController.swift:119:68: error: extra argument 'whereSeparator' in call
                let lines = annotation.subtitle?.split(whereSeparator: \.isNewline)
                                                                       ^~~~~~~~~~~
    /Users/me/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/apple_maps_flutter-0.1.2+4/ios/Classes/Annotations/AnnotationController.swift:114:39: error: unterminated string literal
    \(CGFloat(annotation.calloutOffset.y))")
                                          ^
    /Users/me/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/apple_maps_flutter-0.1.2+4/ios/Classes/Annotations/AnnotationController.swift:114:39: error: unterminated string literal
    \(CGFloat(annotation.calloutOffset.y))")
                                          ^
    /Users/me/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/apple_maps_flutter-0.1.2+4/ios/Classes/Annotations/AnnotationController.swift:114:39: error: unterminated string literal
    \(CGFloat(annotation.calloutOffset.y))")
                                          ^
    /Users/me/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/apple_maps_flutter-0.1.2+4/ios/Classes/Annotations/AnnotationController.swift:114:39: error: unterminated string literal
    \(CGFloat(annotation.calloutOffset.y))")
                                          ^
    /Users/me/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/apple_maps_flutter-0.1.2+4/ios/Classes/Annotations/AnnotationController.swift:114:39: error: unterminated string literal
    \(CGFloat(annotation.calloutOffset.y))")
                                          ^
    /Users/me/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/apple_maps_flutter-0.1.2+4/ios/Classes/Annotations/AnnotationController.swift:114:39: error: unterminated string literal
    \(CGFloat(annotation.calloutOffset.y))")
                                          ^
    /Users/me/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/apple_maps_flutter-0.1.2+4/ios/Classes/Annotations/AnnotationController.swift:114:39: error: unterminated string literal
    \(CGFloat(annotation.calloutOffset.y))")
                                          ^
    /Users/me/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/apple_maps_flutter-0.1.2+4/ios/Classes/Annotations/AnnotationController.swift:114:39: error: unterminated string literal
    \(CGFloat(annotation.calloutOffset.y))")
                                          ^
    /Users/me/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/apple_maps_flutter-0.1.2+4/ios/Classes/Annotations/AnnotationController.swift:114:39: error: unterminated string literal
    \(CGFloat(annotation.calloutOffset.y))")
                                          ^
    /Users/me/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/apple_maps_flutter-0.1.2+4/ios/Classes/Annotations/AnnotationController.swift:114:39: error: unterminated string literal
    \(CGFloat(annotation.calloutOffset.y))")
                                          ^
    /Users/me/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/apple_maps_flutter-0.1.2+4/ios/Classes/Annotations/AnnotationController.swift:114:39: error: unterminated string literal
    \(CGFloat(annotation.calloutOffset.y))")
                                          ^
    /Users/me/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/apple_maps_flutter-0.1.2+4/ios/Classes/Annotations/AnnotationController.swift:114:39: error: unterminated string literal
    \(CGFloat(annotation.calloutOffset.y))")
                                          ^
    /Users/me/Developer/flutter/.pub-cache/hosted/pub.dartlang.org/apple_maps_flutter-0.1.2+4/ios/Classes/Annotations/AnnotationController.swift:114:39: error: unterminated string literal
    \(CGFloat(annotation.calloutOffset.y))")
                                          ^
    note: Using new build system
    note: Planning build
    note: Constructing build description

Could not build the application for the simulator.
Error launching application on iPad Air (3rd generation).

```

**Comment by dominikheeb** [comment]
I also have issues building the app with 1.1.2+4

My Stacktrace:
```
/Users/***/development/flutter/.pub-cache/hosted/pub.dartlang.org/apple_maps_flutter-0.1.2+4/ios/Classes/Annotations/AnnotationController.swift:114:39: error: unterminated string literal
    \(CGFloat(annotation.calloutOffset.y))")
                                          ^
    /Users/***/development/flutter/.pub-cache/hosted/pub.dartlang.org/apple_maps_flutter-0.1.2+4/ios/Classes/Annotations/AnnotationController.swift:114:2: error: invalid component of Swift key path
    \(CGFloat(annotation.calloutOffset.y))")
     ^
    /Users/***/development/flutter/.pub-cache/hosted/pub.dartlang.org/apple_maps_flutter-0.1.2+4/ios/Classes/Annotations/AnnotationController.swift:114:1: error: cannot infer key path type from context; consider explicitly specifying a root type
    \(CGFloat(annotation.calloutOffset.y))")
    ^
```

**Comment by LuisThein** [comment]
Hey, sorry about the inconvenience, should be fixed with version `0.1.9+4`.
